### PR TITLE
added florida hospital scraper -- includes parent class for Tableau downloads

### PR DESCRIPTION
**Pull request by sglyon**
@cc7768 this is preliminary and needs some help... but it does get the file and downloads it

### CODE REVIEW

**Comment by sglyon** [comment]
@cc7768 this has been cleaned up and is ready for review (pending passing tests)

**Comment by sglyon** [comment]
ping @cc7768

File: src/cmdc_tools/datasets/puppet.py [hunk 1]
```diff
@@ -10,12 +19,75 @@
...
+                await button.click()
+
+                # find and click crosstab option to initiate download
+                crosstab = await page.waitForXPath(
+                    f"//button[{self.crosstab_button_selector}]"
+                )
+                await crosstab.click()
+
+                # TODO: better way to wait for download?
+                await asyncio.sleep(5)
```
Comment (cc7768) [inline src/cmdc_tools/datasets/puppet.py:85]: I'm sure this is ignorant of how the page works but can you not do something above like `page.waitForXPath(...download url)` to wait for it to load up? Does the fact that it's a pop up create trouble?

**Review by cc7768 — APPROVED** [review]
I made a few changes. The biggest of which were:

* Add a new class `FloridaHospitalUsage` and changed name of previous version to `FloridaICUUsage`.
  - This class obtains the hospital beds data (as opposed to ICU beds)
  - This class abstracts some of the column cleaning to a new method
* Changes when the `county` column is created and makes them title case as they're stored in the database.
* Changes how we define "today" for the hospital scrapers -- As mentioned in slack, the `dt` column should reflect the local date because that's what it is for any data that is collected which already includes the date... I publicly maintain with some support from you that time zones shouldn't exist and we should all use UTC time.

File: src/cmdc_tools/datasets/owid/__init__.py [hunk 2]
```diff
@@ -35,7 +35,7 @@ def get(self):
         url = "https://covid.ourworldindata.org/data/owid-covid-data.csv"
         return (
             pd.read_csv(url, parse_dates=["date"])
-            .rename(columns={"date": "dt"})
+            .rename(columns={"date": "dt", "cardiovasc_death_rate": "cvd_death_rate"})
```
Comment (cc7768) [inline src/cmdc_tools/datasets/owid/__init__.py:38]: OWID renamed their cardiovascular death rate variable which was causing tests to break. See [this commit](https://github.com/owid/covid-19-data/commit/fa29d46b2a1fc371b824a55dcc230e915ed2f8a8)

**Pull request merged**